### ISSUE TITLE
extension broken in current vscode-insiders ... 

### ISSUE THREAD

**Issue by AntonioMeireles** [bartosz-antosik/vscode-spellright#441]
FWIW, 

```Version: 1.59.0-insider
Commit: e8a59bc6f920e731b47f0bc72263e82aba8b796e
Date: 2021-07-22T05:13:00.851Z (1 day ago)
Electron: 13.1.7
Chrome: 91.0.4472.124
Node.js: 14.16.0
V8: 9.1.269.36-electron.0
OS: Darwin x64 20.6.0
```
```
[2021-07-23 13:22:27.935] [exthost] [error] Activating extension ban.spellright failed due to an error:
[2021-07-23 13:22:27.935] [exthost] [error] TypeError: Cannot read property 'Spellchecker' of null
    at Object.<anonymous> (/Users/am/.vscode/extensions/ban.spellright-3.0.58/lib/bindings.js:46:25)
    at Module.u._compile (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/loader.js:4:1315)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1155:10)
    at Module.load (internal/modules/cjs/loader.js:982:32)
    at Module._load (internal/modules/cjs/loader.js:823:14)
    at Function.f._load (electron/js2c/asar_bundle.js:5:12913)
    at Function.n._load (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:104:28061)
    at Function.P._load (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:104:24649)
    at Function.c._load (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:94:59602)
    at Module.require (internal/modules/cjs/loader.js:1006:19)
    at v (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/loader.js:4:698)
    at Object.<anonymous> (/Users/am/.vscode/extensions/ban.spellright-3.0.58/src/spellright.js:18:18)
    at Module.u._compile (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/loader.js:4:1315)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1155:10)
    at Module.load (internal/modules/cjs/loader.js:982:32)
    at Module._load (internal/modules/cjs/loader.js:823:14)
    at Function.f._load (electron/js2c/asar_bundle.js:5:12913)
    at Function.n._load (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:104:28061)
    at Function.P._load (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:104:24649)
    at Function.c._load (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:94:59602)
    at Module.require (internal/modules/cjs/loader.js:1006:19)
    at v (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/loader.js:4:698)
    at activate (/Users/am/.vscode/extensions/ban.spellright-3.0.58/src/extension.js:21:24)
    at Function._callActivateOptional (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:90:14919)
    at Function._callActivate (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:90:14592)
    at /Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:90:12789
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async Promise.all (index 0)
    at v.$activate (/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:90:20144)
```

All the best and thanks in advance

**Comment by AntonioMeireles** [comment]
basically, it seems that binaries need to be provided to electron-13.x.

**Comment by AntonioMeireles** [comment]
fwiw,  bellow patch and a rebuild got it to work again on macOS intel. 
 
 ```diff
diff --git a/lib/bin/node-spellchecker/build-macos-intel.sh b/lib/bin/node-spellchecker/build-macos-intel.sh
old mode 100644
new mode 100755
index 031ad3c..bd34424
--- a/lib/bin/node-spellchecker/build-macos-intel.sh
+++ b/lib/bin/node-spellchecker/build-macos-intel.sh
@@ -13,3 +13,6 @@ cp build/Release/spellchecker.node ../spellchecker-darwin-intel-9.1.0-x64.node

 node-gyp rebuild --target=11.1.0 --arch=x64 --dist-url=https://atom.io/download/electron
 cp build/Release/spellchecker.node ../spellchecker-darwin-intel-11.1.0-x64.node
+
+node-gyp rebuild --target=13.1.0 --arch=x64 --dist-url=https://atom.io/download/electron
+cp build/Release/spellchecker.node ../spellchecker-darwin-intel-13.1.0-x64.node
\ No newline at end of file
diff --git a/lib/bin/node-spellchecker/package-lock.json b/lib/bin/node-spellchecker/package-lock.json
index 333b787..b894d5e 100644
--- a/lib/bin/node-spellchecker/package-lock.json
+++ b/lib/bin/node-spellchecker/package-lock.json
@@ -271,9 +271,9 @@
       "dev": true
     },
     "nan": {
-      "version": "2.14.0",
-      "resolved": "https://registry.npmjs.org/nan/-/nan-2.14.0.tgz",
-      "integrity": "sha512-INOFj37C7k3AfaNTtX8RhsTw7qRy7eLET14cROi9+5HAVbbHuIWUHEauBv5qT4Av2tWasiTY1Jw6puUNqRJXQg=="
+      "version": "2.14.2",
+      "resolved": "https://registry.npmjs.org/nan/-/nan-2.14.2.tgz",
+      "integrity": "sha512-M2ufzIiINKCuDfBSAUr1vWQ+vuVcA9kqx8JJUsbQi6yf1uGRyb7HfpdfUr5qLXf3B/t8dPvcjhKMmlfnP47EzQ=="
     },
     "once": {
       "version": "1.4.0",
```

**Comment by OmriHarary** [comment]
Broken now with 1.59 Stable, also

**Comment by klausj1** [comment]
Fixed in 3.0.60 (see #443), this issue can be closed.
